selftests/unit/test_utils_params.py

```diff
@@ -86,6 +87,64 @@ def testGetItemMissing(self):
     def testGetItem(self):
         self.assertEqual(self.params['image_size'], "10G")
 
+    def testGetBoolean(self):
+        self.params["foo1"] = "yes"
+        self.params["foo2"] = "no"
+        self.params["foo3"] = "on"
+        self.params["foo4"] = "off"
+        self.params["foo5"] = "true"
+        self.params["foo6"] = "false"
+        self.assertEqual(True, self.params.get_boolean('foo1'))
+        self.assertEqual(False, self.params.get_boolean('foo2'))
+        self.assertEqual(True, self.params.get_boolean('foo3'))
+        self.assertEqual(False, self.params.get_boolean('foo4'))
+        self.assertEqual(True, self.params.get_boolean('foo5'))
+        self.assertEqual(False, self.params.get_boolean('foo6'))
+        self.assertEqual(False, self.params.get_boolean('notgiven'))
+
+    def testGetNumeric(self):
+        self.params["something"] = "7"
+        self.params["foobar"] = 11
+        self.params["barsome"] = 13.0
+        self.assertEqual(7, self.params.get_numeric('something'))
+        self.assertEqual(11, self.params.get_numeric('foobar'))
+        self.assertEqual(11.0, self.params.get_numeric('foobar'), float)
+        self.assertEqual(13, self.params.get_numeric('barsome'))
+        self.assertEqual(13.0, self.params.get_numeric('barsome'), float)
+        self.assertEqual(17, self.params.get_numeric('joke', 17))
+        self.assertEqual(17.0, self.params.get_numeric('joke', 17), float)
```

Comment:
Well 17.0 equals 17, to be sure we'd have to add float test (eg. 13.1) or check for returned type. Anyway it's not critical...

Reply:
Does this look better?:

```python
    def testGetNumeric(self):
        self.params["something"] = "7"
        self.params["foobar"] = 11
        self.params["barsome"] = 13.17
        self.assertEqual(7, self.params.get_numeric('something'))
        self.assertEqual(7, self.params.get_numeric('something'), int)
        self.assertEqual(7.0, self.params.get_numeric('something'), float)
        self.assertEqual(11, self.params.get_numeric('foobar'))
        self.assertEqual(11, self.params.get_numeric('something'), int)
        self.assertEqual(11.0, self.params.get_numeric('foobar'), float)
        self.assertEqual(13, self.params.get_numeric('barsome'))
        self.assertEqual(13, self.params.get_numeric('barsome'), int)
        self.assertEqual(13.17, self.params.get_numeric('barsome'), float)
        self.assertEqual(17, self.params.get_numeric('joke', 17))
        self.assertEqual(17.13, self.params.get_numeric('joke', 17.13), float)
```

Reply:
yep

Reply:
Great, will push the PR in about 2 hours after some minimal testing gives me green light.